Logos/cisco-logo-72px.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <desc>Created with Sketch.</desc>
+    <g id="cisco_sentinel_connector_logo_svg" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="cisco_sentinel_connector_logo" transform="translate(0.000000, 15.000000)" fill="#049FD9">
```

Comment:
Please provide guids so that these are uniquely identifiable across all Azure logo assets

Reply:
Completed